tests/system/server.go

```diff
@@ -63,6 +63,7 @@ func startAKServer(t *testing.T, ctx context.Context, akPath string, userCfg map
 		"http.addr_filename":                  serverHTTPAddrFile, // In the test's temporary directory.
 		"authhttpmiddleware.use_default_user": "false",
 		"svc.seed_objects_path":               seedObjectsPath,
+		"db.debug":                            "true",
```

Comment:
No, this is too noisy for normal system tests, which are too noisy already. It's ok to use it in local testing, not in CI.

Reply:
fixed